extension.neon

```diff
@@ -20,10 +20,14 @@ services:
 			- phpstan.typeSpecifier.functionTypeSpecifyingExtension
 	-
 		class: PHPStan\Type\PHPUnit\Assert\AssertMethodTypeSpecifyingExtension
+		arguments:
+			classWithAssertionMethods: PHPUnit\Framework\TestCase
```

Comment:
I've retained the class used in https://github.com/phpstan/phpstan-phpunit/pull/65/files#diff-afc1ebcbfd95543266d01a244f0e43deL27, but shouldn't be this `PHPUnit\Framework\Assert` too?